pkg/apis/serving/helpers/contexts.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2021 The Knative Authors
```

Comment:
Guess this is 2022 now!
```suggestion
Copyright 2022 The Knative Authors
```